server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/EventDeadLettersIntegrationTest.java

```diff
@@ -333,12 +369,18 @@ public void failedEventShouldBeCorrectlyProcessedByListenerAfterSuccessfulRedeli
         awaitAtMostTenSeconds.until(() -> retryEventsListener.getSuccessfulEvents().size() == 1);
     }
 
+    private void waitForCalls(int count) {
+        calmlyAwait.atMost(ONE_MINUTE).until(() -> retryEventsListener.totalCalls.intValue() >= count);
+    }
+
     @Test
     public void taskShouldBeCompletedAfterSuccessfulGroupRedelivery() {
-        retryEventsListener.setRetriesBeforeSuccess(4);
+        retryEventsListener.callsPerEventBeforeSuccess(MAX_RETRIES + 1);
         generateInitialEvent();
         generateSecondEvent();
 
+        waitForCalls((MAX_RETRIES + 1) * 2);
```

Comment:
extract `(MAX_RETRIES + 1) * 2` in a constant

Reply:
how you would name it?

Reply:
`EXPECTED_CALL_TO_LISTENER` ?

Reply:
TBH, it makes things more complicated to read: the intent is in the method name (waitForCalls), EXPECTED_CALL_TO_LISTENER is just another way to say the same thing. And by using a constant, you expect the reader to jump to the definition to understand how many you expect.

In this case, the constant doesn't make sense as it depends on the calls of this specific test. It could be a local variable but once again, it would not add information.

Reply:
Ok